socs/agents/generator/agent.py

```diff
@@ -281,7 +381,7 @@ def acq(self, session, params=None):
 
             session.data = {"fields": {}}
 
-            pm = Pacemaker(1, quantize=True)
+            pm = Pacemaker(.1)
```

Comment:
This is a relatively low sample rate, is there a reason why we want this?

Reply:
I believe Packemaker accepts a sampling frequency, so this is 10 Hz